hyperspy/tests/utils/test_roi.py

```diff
@@ -548,3 +548,71 @@ def test_interactive(self):
         r.x += 5
         sr2 = r(s)
         np.testing.assert_array_equal(sr.data, sr2.data)
+
+    @pytest.mark.parametrize('snap', [True, False, 'default'])
+    def test_interactive_snap(self, snap):
```

Comment:
What additional value does the 'default' option give here? It is changed to `True` in the test and thus does not differ from directly calling the function with `True`.

Reply:
This is to test that the default behaviour is `snap=True`, without setting it explicitly. I agree that this is not really necessary, because it is fairly obvious if the default value in the `interactive` or `add_widget` method is changed in the future! Shall I remove it?

Reply:
Well, I misunderstood how the test works on first sight. You set `snap=True` in the `else` only for the test and not as part of the arguments. You can leave it as is.